src/sklearndf/wrapper/_wrapper.py

```diff
@@ -1211,6 +1220,13 @@ class MetaEstimatorWrapperDF(
     copy of the `X` and `y` parameters, so we need to make sure that these are converted
     back to data frames.
 
+    We also ensure that the embedded estimator is a native sci-kit learn estimator.
```

Comment:
Just started reviewing the PR - only a nitpick here: `sci-kit` to `scikit-learn`

Reply:
Good catch, thanks @mtsokol !